src/app/popups/add-family-member/add-family-member.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { MD_DIALOG_DATA, MdDialog, MdDialogRef } from '@angular/material';
+
+import { ParticipantUpdateResultDialogComponent } from "../../dialogs/participant-update-result-dialog.component";
+
+import {ComponentService} from "../../services/component.service";
+import {DSMService} from "../../services/dsm.service";
+import {RoleService} from "../../services/role.service";
+
+@Component({
+  selector: 'app-add-family-member',
+  templateUrl: './add-family-member.component.html',
+  styleUrls: ['./add-family-member.component.css']
+})
+export class AddFamilyMemberComponent implements OnInit {
+
+  familyMemberFirstName: string;
+  familyMemberLastName: string;
+  familyMemberSubjectId: string;
+  relations = [
```

Comment:
could that info come from field_settings RGP_SUBJECT_STATUS_GROUP.MEMBER_TYPE

Reply:
@SimoneMaiwald maybe statics.ts would more convenient place for it?